package.json

```diff
@@ -47,16 +47,25 @@
     "@testing-library/cypress": "^6.0.0",
     "@testing-library/jest-dom": "^5.9.0",
     "@testing-library/react": "^10.1.0",
+    "@typescript-eslint/eslint-plugin": "2.x",
+    "@typescript-eslint/parser": "2.x",
     "axe-core": "^3.5.4",
+    "babel-eslint": "10.x",
     "babel-jest": "^26.0.1",
     "babel-preset-gatsby": "^0.4.8",
     "cypress": "^4.7.0",
     "cypress-axe": "^0.8.1",
+    "eslint": "6.x",
     "eslint-config-prettier": "^6.11.0",
     "eslint-config-react-app": "^5.2.1",
     "eslint-plugin-chai-friendly": "^0.6.0",
     "eslint-plugin-cypress": "^2.11.1",
+    "eslint-plugin-flowtype": "4.x",
```

Comment:
Just a note, all the new deps here are all peer deps for `eslint-config-react-app`.